apps/src/javalab/JavalabEditor.jsx

```diff
@@ -24,7 +24,7 @@ import {
 import {DisplayTheme} from './DisplayTheme';
 import PropTypes from 'prop-types';
 import {EditorView} from '@codemirror/view';
-import {editorSetup} from './editorSetup';
+import {editorConfig} from '@cdo/apps/lab2/views/components/editor/editorConfig';
```

Comment:
I'm still confused about when we use absolute vs. relative imports for `apps` code, but that's neither here nor there 🙃 Do you guideline?

Reply:
My rule of thumb is I use relative if it's in the same folder, otherwise absolute. Open to making it all absolute if that makes more sense!

Reply:
As-is is fine. We can update as we go if we ever decide on a team-wide standard, but thanks for sharing what you do!